daemon/algod/api/algod.oas2.json

```diff
@@ -1209,7 +1209,7 @@
       "post": {
         "tags": [
           "public",
-          "experimental"
+          "nonparticipating"
```

Comment:
I chose nonparticipating over participating because the endpoint only reads state from the ledger about past blocks. So this fits better with the other nonparticipating endpoints, which are all "read-only" like this